sktime/transformations/panel/summarize/_extract.py

```diff
@@ -17,18 +22,53 @@
 class PlateauFinder(BaseTransformer):
     """Plateau finder transformer.
 
-    Transformer that finds segments of the same given value, plateau in
-    the time series, and
-    returns the starting indices and lengths.
+    Transformer that finds segments of the same given value, aka plateau(s) in
+    the time series, and returns the starting indices and lengths. It expects a
+    dataframe with one column of nested Series and it extracts the plateau for
+    each series. The output is a dataframe with two columns of nested lists,
+    where the first column has the lists with the starting indices of the
+    plateau(s), and the second column has the lists of the length of the
+    plateau(s). If no flate segements are found, the output is a dataframe of
+    nested empty lists.
+
+    In this context the `value` parameter tells the transformer to look for flat
+    segments with a specific value, the `min_lenght` tells to filter
```

Comment:
is it always plateaus of that given value?

Reply:
yep. If the value is not given it is set to `NaN`